scripts/src/primazactl/primazactl.py

```diff
@@ -0,0 +1,101 @@
+import argparse
+import os
+import semver
+import sys
+from pathlib import Path
+from primazactl.primazamain import primazamain
+sys.path.append('./scripts/src/')
```

Comment:
is this needed?

Reply:
oops, no. it was when I was running using python3 but not now.

Reply:
```suggestion
```